src/components/AttachmentPicker/index.js

```diff
@@ -35,14 +34,9 @@ function AttachmentPicker(props) {
                 type="file"
                 ref={fileInput}
                 onChange={(e) => {
-                    let file = e.target.files[0];
+                    const file = e.target.files[0];
 
                     if (file) {
-                        const cleanName = FileUtils.cleanFileName(file.name);
-                        if (file.name !== cleanName) {
-                            file = new File([file], cleanName);
-                        }
-                        file.uri = URL.createObjectURL(file);
```

Comment:
We forgot to attach the .uri property back above

Reply:
@Li357 You are right it has caused regression, and we fixed it in another PR - https://github.com/Expensify/App/pull/22903